src/python-common/ceph/rgw/rgwam_core.py

```diff
@@ -193,46 +190,54 @@ def __init__(self, env: EnvArgs):
         self.env = env
 
     def list(self):
-        ze = ZoneEnv(self.env)
-
-        params = ['realm',
-                  'list']
-
-        return RGWAdminJSONCmd(ze).run(params)
+        try:
+            ze = ZoneEnv(self.env)
```

Comment:
Knit: why is this type called `ZoneEnv` not `MultisiteEnv`? I feel like that would be a more suitable name given how this type is used in all calls of `RGWAdminJSONCmd`

Reply:
This is legacy code, I can rename the variable as part of other PR to not introduce a lot of noise on this one.